src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12764,10 +12764,7 @@ Unit* Creature::SelectVictim()
         return NULL;
 
     if (target && _IsTargetAcceptable(target) && CanCreatureAttack(target))
-    {
-        SetInFront(target);
         return target;
```

Comment:
does it work without trouble? is serverside orientation still updated correctly?